toolkits/global/packages/global-autocomplete/__tests__/index.js

```diff
@@ -1,219 +1,253 @@
 'use strict';
```

Comment:
The changes in the test here are largely just rejigging, removing duplication etc. I've added a new test block from line 193.